### PR TITLE
Es6 implementation

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request features a transformation of the original AwesomeBooks project (found here https://github.com/mrjuda/books) into a new project featuring ES6 syntax. All of the javascript classes & objects have been transformed into modules and set up via import/export in the main JS file for this website.

This site also features a DateTime object from the Luxon library. Upon loading, the user will have their local time displayed at the top right corner of the page beneath the navbar.

### CODE REVIEW

File: modules/library.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  pullFromStorage = () => {
+    const parsed = JSON.parse(localStorage.getItem('strShelf'));
+    let counter = this.shelf.length;
+    const preShelf = [];
+    for (let i = 0; i < this.shelf.length; i += 1) {
+      const parsedBook = parsed[`${counter}`];
+      const tempShelf = Library.displayBook(parsedBook.title, parsedBook.author, counter);
+      preShelf.unshift(tempShelf);
+      counter -= 1;
+    }
```
Comment (ShoiraTa) [inline modules/library.js:40]: - [x] Data is not being displayed from the local storage, please check this video: 

https://user-images.githubusercontent.com/77038610/159342287-aa34aa55-c463-45b8-a7c9-22a28c81056d.mp4

- [x] Make sure that after page refresh the all the data from localStorage is displayed on Library page

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @AKeeganDev,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

- [x] Please consider adding more sections in README file like `Inspiration`, `license`, getting started 

_Check the comments under the review._ 

## Optional suggestions

- [ ] OPTIONAL: Please consider adding more links in Authors section of `Aaron Keegan`

- [ ] OPTIONAL: Please consider breaking your code into more models with their own methods inside. 
- [ ] OPTIONAL:  Please consider updating readme with the live link https://akeegandev.github.io/Awesome_Books/ 😉 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/library.js (hunk 1 above)
Comment (AKeeganDev) [reply modules/library.js:40]: @ShoiraTa  Last week, my partner (who owned this code) received an email saying that this requirement was going to be optional. Is there a way that can continue to be honored?
I feel this exercise today was more based on ES6 and using Luxon where this requirement feels like something that should have been made required last week. If we can negotiate that I would appreciate it. If not I will start working on it. Sorry to contest your requirement. Thank you!

Comment (ShoiraTa) [reply modules/library.js:40]: Hi @AKeeganDev, I understand your condition, but one of the requirements is to check if app is working as expected. Try to do best to resolve this issue. I can check tomorrow what might be causing this issue and help if needed 👍🏻 
I'm sure there is a small error causing this error, try to find it, if can't please contact me in slack, my user name is @shoira

**Review by Meltrust — APPROVED** [review]
Hi @AKeeganDev,


## Wow, you did it 🎉
Brilliant
## Thank you for ensuring that after page refresh all the data from localStorage is displayed on the Library page, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AKeeganDev** [comment]
> Hi @AKeeganDev,
> 
> ## Wow, you did it 
> Brilliant
> 
> ## Thank you for ensuring that after page refresh all the data from localStorage is displayed on the Library page, as requested by the previous reviewer   
> ### Well done!
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@Meltrust  Thank you!

**Pull request merged**